pkg/collector/py/datadog_agent.c

```diff
@@ -181,8 +180,8 @@ static PyObject *add_external_tags(PyObject *self, PyObject *args) {
         }
 
```

Comment:
part of solution to below is should have
`memset(tags, 0, sizeof(char*) * tags_len)` 
here.